package.json

```diff
@@ -24,5 +24,8 @@
     "format:fix": "prettier --write 'packages/*/(src|test)/**/*.(mts|ts)' 'integration-tests/*/*.(mts|ts)'",
     "ci:typecheck": "turbo run ci:typecheck",
     "ci:test": "turbo run ci:test"
+  },
+  "overrides": {
+    "@types/react": "npm:types-react@rc"
   }
```

Comment:
this is recommended for the react types to be found correctly, at least while React 19 is not marked stable. for pnpm, this must be in the root of the workspace; i'm not actually sure if the same is true for yarn.